test/gateway/eip20_cogateway/confirm_stake_intent.js

```diff
@@ -275,25 +275,6 @@ contract('EIP20CoGateway.confirmStakeIntent() ', (accounts) => {
     );
   });
 
-  it('should fail when staker nonce is already consumed.', async () => {
```

Comment:
Why did you delete this test? Shouldn't it still be tested? (e.g. try to use nonce `0` twice in a row)

Reply:
This test is already covered in `should fail to confirm stake intent if its already confirmed once`.
In fact, this test was invalid as the staker nonce always started with 1.

Reply:
ok